src/Pmi/Controller/DefaultController.php

```diff
@@ -571,4 +572,34 @@ public function hideTZWarningAction(Application $app, Request $request)
         $request->getSession()->set('hideTZWarning', true);
         return (new JsonResponse())->setData([]);
     }
+
+    public function mockLoginAction(Application $app, Request $request)
+    {
+        if (!$app->isLocal() && !$app->getConfig('gae_auth')){
+            return $app->abort(403);
+        }
+        $loginForm = $app['form.factory']->createNamedBuilder('login', FormType::class)
```

Comment:
Can we prefill this with a value (`test@example.com` is fine), instead of setting the placeholder? That way we can just click the submit button.